functions/public/profile.hbs

```diff
@@ -84,6 +89,7 @@
         <script src="/__/firebase/init.js"></script>
 
         <script src="scripts/profile.js" type="text/javascript"></script>
+        <script src="scripts/home.js" type="text/javascript"></script>
```

Comment:
Why do we have home.js in the profile page?

Reply:
For the profile picture and username to be displayed on the navigation